File: next-main/src/main/java/com/next/reservations/core/repository/AlternativeRTDateRangeRepository.java
package com.next.reservations.core.repository;

import com.next.reservations.core.domain.AlternativeReservationTimeDateRange;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AlternativeRTDateRangeRepository extends JpaRepository<AlternativeReservationTimeDateRange, Long> {
}

File: next-main/src/main/kotlin/com/next/reservations/core/repository/AlternativeRTDateRangeRepository.kt
package com.next.reservations.core.repository

import com.next.reservations.core.domain.AlternativeReservationTimeDateRange
import org.springframework.data.jpa.repository.JpaRepository

interface AlternativeRTDateRangeRepository : JpaRepository<AlternativeReservationTimeDateRange, Long>{

}
